src/Select/Select.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from 'react'
+import VirtualizedSelect from 'react-select-virtualized'
+import { components } from 'react-select'
+import cx from 'classnames'
+import Notification from '../Notification'
+import Icon from '../Icon'
+import styles from './Select.module.scss'
+
+function SelectContainer (props) {
```

Comment:
For components we are using `const`. It's especially useful for implicit `return`, .e.g `const SelectContainer = props => <div>...</div`. Same is true for components below